tools/manifest-templator/manifest-templator.go

```diff
@@ -254,6 +269,9 @@ func main() {
 	kubevirtIpamControllerImage := flag.String("kubevirt-ipam-controller-image", components.KubevirtIpamControllerImageDefault, "The kubevirtipamcontroller-image managed by CNA")
 	dumpOperatorCRD := flag.Bool("dump-crds", false, "Append operator CRD to bottom of template. Used for csv-generator")
 	inputFile := flag.String("input-file", "", "Not used for csv-generator")
+	clusterDNSNamespace := flag.String("cluster-dns-namespace", "kube-system", "The namespace name where the cluster DNS endpoint reside")
```

Comment:
/hold

i am against it, as discussed offline

we have IsOpenShift all over for binary values (values that can be either k8s set of values or OpenShift
please see render functions

this kind of change involves more repositories and makes the code less self contained, and then we have 2 ways of doing stuff we are doing for long time in one way

Reply:
I removed this part for now, so that this PR only adds network-policies for CNAO.
Supporting other platformes (OCP) will be done in a follow up.
PTAL

Reply:
Thanks, 

btw about the previous comment, cnao must should stay dual as standalone
having that other method require a layer that will customize it, which also may hint this is undesired